backend/plugins/captcha/geetest/lib/GeetestCaptchaService.php

```
@@ -181,6 +190,34 @@ private function scriptResult(string $action, string $content): array
         ];
     }
 
+    /**
+     * 本地代理可能使用自签名根证书，沿用系统 GeeTest TLS 配置；
+     * 生产环境默认仍开启证书校验，也支持通过 CA bundle 指定信任链。
+     *
+     * @param  array<string, mixed>  $config
+     * @return array{verify: bool|string}
+     */
+    private function httpOptions(array $config): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$config'. (undefined)

(UnusedFormalParameter)